apps/meteor/app/cloud/server/functions/saveRegistrationData.ts

```diff
@@ -63,7 +59,10 @@ async function saveRegistrationDataBase({
 		{ _id: 'Cloud_Workspace_Registration_Client_Uri', value: registration_client_uri },
 	];
 
-	const promises = settingsData.map(({ _id, value }) => Settings.updateValueById(_id, value));
+	const promises = [
+		...settingsData.map(({ _id, value }) => Settings.updateValueById(_id, value)),
+		WorkspaceCredentials.updateCredentialByScope('', '', new Date(0)),
```

Comment:
Maybe change this to an object parameter, so it's easier to understand what the empty strings are.